package.json

```diff
@@ -13,5 +13,8 @@
     "elm-tooling": "1.13.0",
     "vite": "4.1.4",
     "vite-plugin-elm": "2.8.0-beta.3"
+  },
+  "dependencies": {
+    "elm-format": "^0.8.6"
```

Comment:
This is already included in the elm-tooling imports https://github.com/geeksforsocialchange/teamwilder/blob/main/elm-tooling.json and also not part of this ticket, so we can remove.